sympy/solvers/ode.py

```diff
@@ -841,6 +842,18 @@ def classify_ode(eq, func=None, dict=False, ics=None, **kwargs):
     :py:mod:`~sympy.solvers.ode` docstring for a list of all supported hints
     that can be returned from :py:meth:`~sympy.solvers.ode.classify_ode`.
 
+    Keyword arguments
+    :prep -- True/False, if true then it preprocess the equality and then generate a function from it,
+    which is _preprocess()
+
+    :n -- no. of terms
+
+    :eta -- Used for the lie_group hint
+
+    :x0 -- Point, to find out a definite power series solution
+
+    :xi -- Used for the lie_group hint
```

Comment:
Please use the numpydoc format: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt